httpd/static/cheezoid_canvas.html

```diff
@@ -30,8 +43,8 @@
                 prevX = currentX
                 prevY = currentY
                 //set new current mouse click
-                currentX = e.clientX - canvas.offsetLeft;
-                currentY = e.clientY - canvas.offsetTop;
+                currentX = e.clientX - canvas.offsetLeft - Number(canvas.style.borderWidth.replace("px",""));
+                currentY = e.clientY - canvas.offsetTop - Number(canvas.style.borderWidth.replace("px",""));
```

Comment:
This works on the canvas, but would be better to test it on the cheezoid before merging.